addons/gamification/data/mail_template_data.xml

```diff
@@ -357,7 +357,7 @@
                 </p>
             </td>
         </tr>
-        <tr t-if="user.signature">
+        <tr t-if="user.signature and not user.has_group('base.group_portal')">
```

Comment:
What is the point of signing as internal user if the user happens to be one?

Shouldn't we simply remove the signature? Or use a "company signature" if there is one defined somewhere?

Reply:
Hello @flch-odoo @LNAodoo 
As per my discussion with lna, the point is to remove the signature only for the portal user because the portal user can never update their signature so it's relevant to have a signature for them.
Internal user can update their signature from the preference section.
I agree that there is no need for any signature in the template, we can ask 
@LNAodoo for this.

Reply:
Agree that it is more an "informational" template, so not sure current user really matters. Could be better a generic signature like "sent by mycompany" :)

Reply:
Hello @flch-odoo 
Not sure if we need to use the user's login company or the company name mentioned in the res.partner model

Reply:
I saw Luc's message on task. Indeed when granting from a user to another user, it is better to know who gave the badge. When the user is portal / inactive (robodoo for example), would fallback on user's company (the 'working' company, like Odoo SA). We could have something more complete like
```
if not user.share and user.active:
    // insert user's signature
else:
    // insert user.company_id information
```

Reply:
active should be in first condition actually

Reply:
Hello @tde-banana-odoo 
I have made the changes.